theme/partials/ui/image.html.twig

```diff
@@ -66,7 +66,7 @@
     <template x-if="!shouldLoadImages">
         {% embed 'partials/ui/placeholder' %}
                 {% block data %}
-                    isLoading: !shouldLoadImages,
+                    isLoading: true,
```

Comment:
to clarify, I think the watcher (and possibly the whole `init` function) is unnecessary